gtsfm/scene_optimizer.py

```diff
@@ -313,13 +324,14 @@ def save_metrics_reports(metrics_graph_list: Delayed) -> List[Delayed]:
 
 
 def save_full_frontend_metrics(
-    two_view_report_dict: Dict[Tuple[int, int], TwoViewEstimationReport], images: List[Image]
+    two_view_report_dict: Dict[Tuple[int, int], TwoViewEstimationReport], images: List[Image], save_fname: str
```

Comment:
super nit: the function already has _save_ in it, so "filename" is sufficient for the argument name.

Reply:
good point, updated now